scripts/release-prompts.ts

```diff
@@ -144,8 +144,8 @@ export async function promptRelease(opts: BuildOptions): Promise<NormalizedRelea
  * answers is to be used.
  *
  * @param answers user provided answers to pick from
- * @returns the tag to use. defaults to 'UNKNOWN' if the tag cannot be determined.
+ * @returns the tag to use. defaults to `null` if no tag was specified
  */
-export function determineAnsweredTagToUse(answers: ReleasePromptAnswers): string {
-  return answers.tag ? answers.tag : answers.specifiedTag ? answers.specifiedTag : 'UNKNOWN';
+export function determineAnsweredTagToUse(answers: ReleasePromptAnswers): string | null {
+  return answers.specifiedTag ? answers.specifiedTag : answers.tag ? answers.tag : null;
```

Comment:
would it possibly be clearer to do `answers.specifiedTag ?? answers.tag ?? null` ? just a little less syntax noise that way

Reply:
@tanner-reits actually asked that the first time around too 😆 The reason we don't do that is the nullish coalescing won't handle empty strings

Reply:
ahh right - if we want `""` to be falsy we might get away with `answers.specifiedTag || answers.tag || null` then

but we can also just keep as-is!

Reply:
Nah - I like that better. I'm all in favor of simplifying this - done in https://github.com/ionic-team/stencil/pull/4257/commits/29d4f190ec40ebcf79dd1ffc8f8678edbac8741f